Build/Module.xml

```diff
@@ -12,5 +12,9 @@
     <Package Uri="https-git://github.com/OutOfOrder/MonoKickstart.git" Folder="ThirdParty/Kickstart" GitRef="master" />
```

Comment:
I think GenerateNuGetRepositiories should be true here. I'm trying to set up NUnit 3.5.0 with the NuGet package and I had to change this. @dellis1972